src/garage/torch/q_functions/continuous_mlp_q_function.py

```diff
@@ -0,0 +1,40 @@
+"""This modules creates a continuous MLP Q fcuntion network."""
+
+
+class ContinuousMLPQFunction:
```

Comment:
Like above, please rename MLP to NN.

Reply:
Can we also make this a module so that we don't need to do something like `qf.module(observations, actions)`? Instead, we can do `qf(observations, actions)` to build a graph?

Reply:
i agree. this is the design we are working towards in TF